src/main.cpp

```diff
@@ -227,46 +227,46 @@ int main(int argc, char* argv[])
                                                      "less likely to be useful for normal use of this application:\n")
                  .arg(QLatin1String(APP_TARGET));
         // From documentation and from http://qt-project.org/doc/qt-5/qapplication.html:
-        texts << QCoreApplication::translate("main", "       --dograb        ignore any implicit or explicit -nograb.\n"
-                                                     "                       --dograb wins over --nograb even when --nograb is last on\n"
-                                                     "                       the command line.\n");
+        texts << QStringLiteral("       --dograb        ignore any implicit or explicit -nograb.\n"
+                                "                       --dograb wins over --nograb even when --nograb is last on\n"
+                                "                       the command line.\n");
 #if defined(Q_OS_LINUX)
         // Need to split these into actually seperate strings, as Crowdin translation would not split strings on #if otherwise.
```

Comment:
You'll want to remove this comment as well.

Reply:
Feedback applied.